File: go_blockchain/one_basic_prototype/blc/blockchain.go
package blc

type Blockchain struct {
	//存储有序区块
	Blocks []*Block
}

//1.创建带有创世区块的区块链
func CreateBlockchainWithGenesisBlock() *Blockchain {
	//创建创世区块
	genesisBlock := CreateGenesisBlock("介系一个船新的创系区块")
	//返回链对象
	return &Blockchain{[]*Block{genesisBlock}}
}

File: java_blockchain/one_basic_prototype/src/main/java/com/whoiszxl/blc/model/po/Blockchain.java
package com.whoiszxl.blc.model.po;

import java.util.LinkedList;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Builder
public class Blockchain {

	private LinkedList<Block> blocks;
}
